pkg/process/tagger/tagger.go

```diff
@@ -82,9 +82,9 @@ func findCandidateTags(appVersion *runtime.AuroraVersion, outputRepository strin
 	} else {
```

Comment:
Siden if er så langt oppe (og litt rart indentert?), kanskje legge på en kommentar?
```suggestion
	} else {
	// this is a snapshot version
```